ai_genomics/analysis/data_scoping/patent_data/search_evaluation.md

```diff
@@ -0,0 +1,64 @@
+
+|        | Not Genomics | Genomics |
+| :----- | -----------: | -------: |
+| Not AI |            0 |       21 |
+| AI     |           11 |       23 |
+
+### Suggested improvements
+
+First we should refine the IPC/CPC patent codes used for our definition of genomics. This could be done manually (e.g. by pruning topics such as 'genetic algorithms') or by using a more sophisticated for creating the definition (e.g. embeddings instead of keywords). As the first method is simpler, this should be attempted first. We should also ensure the IPC/CPC codes used are as expansive as possible to make sure that we also capture genomics related topics that might not have appeard in the results from this query. A combined keyword search and manual filtering approach is probably suitable for this.
+
+Second, we should also refine the columns used from the USPTO data for our definition of AI. We could drop AI hardware, planning and control and knowledge processing to be left with machine learning, natural language processing, computer vision evolutionary computation and speech.
+
+## Approach 2
+
+### Collection
+
+The same approach as above is used, except that the USPTO data is filtered by specific AI flags, rather than for all 'AI related' work. These are machine learning, evolutionary computation, computer vision, speech and natural language processing (using the columns prefixed by `predict50_`). This yielded ~3.5k patents.
+
+### Evaluation
+
+In this case, a random sample of 50 patents (`outputs/data/patents/ai_genomics_patents_ml_evo_cv_nlp_sp_verified.csv`) were evaluated in the same way described above. As a proportion, we see improved recall on patents related to AI, but a decrease in those related to genomics. Annecdotally, there is a high representation of patents describing genetic algorithms, but with no relationship to genomics.
+
+There is also some noise in the USPTO data resulting in false positives, which contribute to the 20 patents assessed as _Not AI_.
+
+|        | Not Genomics | Genomics |
+| :----- | -----------: | -------: |
+| Not AI |            6 |       14 |
+| AI     |           19 |       11 |
+
+### Suggested improvements
+
+- The next step would be to expand and refine the patent code definition of genomics and re-run the evaluation before advancing on to any more sophisticated methods.
+- Another step would be to further refine the data in the USPTO AI patents by setting a higher threshold on the prediction probability. The `predict50_` fields set a minimum threshold of 0.5 on the prediction probability for a patent to be classed as AI related. Setting this higher (e.g. 0.9) would eliminate some false positives. A larger sample of labelled data would be needed to determine whether the benefits gained in precision would offset the loss in recall.
+
+## Next steps
```

Comment:
really helpful next steps - even when we set threshold to 0.95 (resulting in approx. 1500 patents total) and removed all genetic algorithms related IPC and CPC codes, we found only 52% of patents where talking about AI AND genomics